CHANGES.md

```diff
@@ -1,48 +1,130 @@
-### 2.0.0
+### unreleased
 
 **irmin-unix**
 
-- Added `Cli` module to expose some methods to simplify building command-line
-  interfaces using Irmin (517, @zshipko)
-- Switched from custom configuration format to YAML (#504, @zshipko)
-- Added global config file (`~/.irmin/config.yml`)
-- Fixed parsing of commit hashes in `revert` command (#496, @zshipko)
+- Add `Cli` module to expose some methods to simplify building command-line
+  interfaces using Irmin. (517, @zshipko)
+
+- Switch from custom configuration format to YAML. (#504, @zshipko)
+
+- Add global config file. (`~/.irmin/config.yml`)
```

Comment:
is there an env var that can override this? worth noting here if so

Reply:
From the manpages:

> The irmin.yml is read by default if it is found in the current working directory or defined globally as $HOME/.irmin/config.yml. The configuration file path can also be set using the --config command-line flag.

I'll add some detail here to make this entry more helpful.